docs/staking-and-delegation/delegation.md

```diff
@@ -86,27 +92,46 @@ You will get an output like this (click on the image to zoom):
 
 See below for an explanation of the column headings in the above terminal output:
 
-| Column | Meaning                                                 |
-| :------ | ------------------------------------------------------------|
-| INDEX   | Delegates with larger total stake are higher in the list. |
-| DELEGATE | The name of the delegate. Click on the name to visit the delegate website. Only shows if the delegate has registered. |
-| SS58    | The [SS58 hotkey of the delegate](../getting-started/wallets#list-all-the-local-wallets).                       |
-| NOMINATORS    | The number of nominators, i.e., delegators, who have delegated to this delegate. This is also the number of unique cold keys (i.e., number of nominators) who have nominated **to** this hotkey (i.e., to this delegate).                       |
-| DELEGATE STAKE(τ)    | The amount of delegate's own stake (not the TAO delegated from any nominators). This is the the amount of stake that the delegate-owned coldkey has delegated to this delegate's hotkey (distinct from TAO delegated from others).                       |
-| TOTAL STAKE(τ)    | The total amount of stake delegated to this delegator's hotkey.                       |
-| CHANGE/(4h)     | The percent change in the total stake delegated to this delegate within the past 4 hours.                       |                   |
-| VPERMIT    | Shows the subnets for which the validator permits are held by the delegate. 
-| TAKE    | Shows the delegate take percentage.                      |
-| NOMINATOR/(24h)/kτ    | Stake reward distributed to this delegate's nominators within the past 24 hour period (per 1000 TAO). |
-| DELEGATE/(24h)    | Stake reward cut taken by this delegate within the past 24 hour period.                        |
-| Desc    | A description of the delegate.                     |
+| Column                | Meaning                                                                                                                                               |
+| :------               | ------------------------------------------------------------                                                                                          |
+| INDEX                 | Delegates with a more significant total stake are higher on the list.                                                                                 |
```

Comment:
I think "with a larger total stake" reads better. Because "significant" is context-dependent and it does not say large or small. We want to actually mean "large" as in numerically large amount of stake.